Project-Hotpot/Project-Hotpot/Managers/SpotifyManager.swift

```diff
@@ -134,6 +146,33 @@ extension SpotifyManager: SPTSessionManagerDelegate {
 
 // MARK: - Networking
 extension SpotifyManager {
+    func fetchGenreSeeds(completion: @escaping ([String: Any]?, Error?) -> Void){
+        guard let url = URL(string: "https://api.spotify.com/v1/recommendations/available-genre-seeds"),
+              let accessToken = self.appRemote.connectionParameters.accessToken
+        else {
+            NSLog("accessToken is nil, returning early")
+            return
```

Comment:
You'll likely want to call the completion with some sort of error to let the app know "I didn't have an access token, I need you to fetch one for the next call".